README.adoc

```diff
@@ -126,9 +164,7 @@ include::finish/pom.xml[]
 ----
 
 The [hotspot file=0]`pom.xml` file starts with a root [hotspot=project file=0]`<project/>` 
-element and [hotspot=modelVersion file=0]`<modelversion/>`, which is always set to `4.0.0`. The [hotspot=parent file=0]`<parent/>` section 
-of the POM specifies that Maven will use the https://github.com/WASdev/ci.maven/blob/master/liberty-maven-app-parent/pom.xml[Liberty Maven parent POM^] 
-to build the project. This is the minimum you technically need for the POM to be valid. 
+element and [hotspot=modelVersion file=0]`<modelversion/>`, which is always set to `4.0.0`. 
```

Comment:
Change
"The [hotspot file=0]`pom.xml` file starts with a root [hotspot=project file=0]`<project/>` element and [hotspot=modelVersion file=0]`<modelversion/>`, which is always set to `4.0.0`. "
To
"The [hotspot file=0]`pom.xml` file starts with a root [hotspot=project file=0]`<project/>` element and a [hotspot=modelVersion file=0]`<modelversion/>` element, which is always set to `4.0.0`. "